src/scriptdev2/scripts/eastern_kingdoms/blasted_lands.cpp

```diff
@@ -27,6 +27,114 @@ EndContentData */
 
 #include "precompiled.h"
 
+enum
+{
+	NPC_RAZELIKH = 7668,
+	NPC_SEVINE = 7671,
+	NPC_ALLISTARJ = 7670,
+	NPC_GROL = 7669,
+
+	GOB_RAZELIKH = 141812,
+	GOB_SEVINE = 141859,
+	GOB_ALLISTARJ = 141858,
+	GOB_GROL = 141857,
+
+	SPELL_SELFSTUN = 9032
+};
+
+bool m_uiStoneDestroyed = false;
+bool m_uiStoneEnabled = false;
+GameObject* m_uiStoneGameObject;
+uint32 m_uiStunTimer = 0;
+
+struct npc_stone_servantAI : public ScriptedAI
+{
+	npc_stone_servantAI(Creature* pCreature) : ScriptedAI(pCreature) { Reset(); }
+
+	void Reset() override
+	{
+		m_uiStoneDestroyed	= false;
+		m_uiStoneEnabled	= false;
+		m_uiStunTimer = 0;
+	}
+
+	GameObject* FindStoneOfBinding()
+	{
+		//Find nearest Stone of Binding object with an entry corresponding to the NPC entry.
+		switch (m_creature->GetEntry())
+		{
+		case NPC_RAZELIKH:
+			return GetClosestGameObjectWithEntry(m_creature, GOB_RAZELIKH, 40.0f);
+			break;
+		case NPC_SEVINE:
+			return GetClosestGameObjectWithEntry(m_creature, GOB_SEVINE, 40.0f);
+			break;
+		case NPC_ALLISTARJ:
+			return GetClosestGameObjectWithEntry(m_creature, GOB_ALLISTARJ, 40.0f);
+			break;
+		case NPC_GROL:
+			return GetClosestGameObjectWithEntry(m_creature, GOB_GROL, 40.0f);
+			break;
+		default:
+			return nullptr;
+			break;
+		}
+	}
+
+	void DamageTaken(Unit* /*pDoneBy*/, uint32& uiDamage) override
+	{
+		if (uiDamage < m_creature->GetHealth())
+			return;
+
+		if(m_creature->GetHealth() >= 1)
+			m_creature->SetHealth((uint32)1);
+
+		if (!m_uiStoneDestroyed)
+		{
```

Comment:
these brackets could also be omitted since the contents is a one liner, just like you've done above.